config/dynamic.toml

```diff
@@ -0,0 +1,597 @@
+repeatable = 0
+minimum_required_age = 0
+
+["Light Pirates"]
+weight = 3
+min_pop = 15
+blacklisted_roles = []
+min_antag_cap = 0
+# max_antag_cap = min_antag_cap
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Heavy Pirates"]
+weight = 3
+min_pop = 25
+blacklisted_roles = []
+min_antag_cap = 0
+# max_antag_cap = min_antag_cap
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Midround Wizard"]
+weight.1 = 0
+weight.2 = 0
+weight.3 = 1
+weight.4 = 2
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Midround Nukeops"]
+weight.1 = 0
+weight.2 = 1
+weight.3 = 3
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap.denominator = 18
+min_antag_cap.offset = 1
+# max_antag_cap = min_antag_cap
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Midround Clownops"]
+weight = 0
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap.denominator = 18
+min_antag_cap.offset = 1
+# max_antag_cap = min_antag_cap
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Blob"]
+weight.1 = 0
+weight.2 = 1
+weight.3 = 3
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Xenomorph"]
+weight.1 = 0
+weight.2 = 1
+weight.3 = 5
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Nightmare"]
+weight = 5
+min_pop = 15
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Space Dragon"]
+weight.1 = 0
+weight.2 = 3
+weight.3 = 5
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Abductors"]
+weight = 5
+min_pop = 20
+blacklisted_roles = []
+min_antag_cap = 2
+# max_antag_cap = min_antag_cap
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Space Ninja"]
+weight.1 = 0
+weight.2 = 0
+weight.3 = 1
+weight.4 = 2
+min_pop = 30
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Revenant"]
+weight = 5
+min_pop = 10
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Midround Changeling"]
+weight = 5
+min_pop = 15
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Paradox Clone"]
+weight = 5
+min_pop = 10
+blacklisted_roles = []
+min_antag_cap = 1
+max_antag_cap = 1
+repeatable_weight_decrease = 2
+repeatable = 0
+minimum_required_age = 0
+
+["Voidwalker"]
+weight = 5
+min_pop = 15
```

Comment:
This was 40 minimum pop because there are only so many people near or in space in a round :/